### DISCUSSION TITLE
Can we install ONLY for the current user & NOT for all other users on the win11 system?

### DISCUSSION THREAD

**Discussion by subanana** [valinet/ExplorerPatcher#3487] [Q&A]
Experts plz clarify
Can we install ONLY for the current user & NOT for all other users on the win11 system? tkz !

**Comment by pyrates999** [comment]
No, sorry.  EP is installed for all users right now as it requires the user to be a member of the administrators group, and then elevated privileges to install, update, and uninstall.

**Reply by deletedpaula1p-cloud** [reply]
but i want it on my school i cant do it without uac

**Reply by pyrates999** [reply]
UAC is a technical requirement to run EP, sorry.

**Reply by deletedpaula1p-cloud** [reply]
dang it
my  school does not let me use uac at all at home i can just do it. WHY??????/

**Reply by deletedpaula1p-cloud** [reply]
dats kinda stopid

**Reply by PeteBeet** [reply]
I can totally live with that! Thanks indeed for the project!

**Reply by deletedpaula1p-cloud** [reply]
also my school banned theese simalar programs no uac now cuz it broke 3 PCs, and this program it is a worm too  slows down pcs
